.github/workflows/comment_commands.yml

```diff
@@ -0,0 +1,164 @@
+name: Comment Commands to Trigger CI
+on:
+  issue_comment:
+    types: created
+
+permissions:
+  checks: write
+
+jobs:
+  pandas_nightly:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: (github.event.issue.pull_request) && github.event.comment.body == '/pandas_nightly'
```

Comment:
Should probably also document those commands

Reply:
@Dr-Irv Do you prefer to have separate commands (pandas nightly for pytest; mypy nightly; pyright strict) or combine them? Would reduce the common setup steps if we combine them.

Reply:
I was thinking of combining the commands into one since we can only trigger one set of tests by command, just trying to figure out how to make a clean combination.

Reply:
If by "commands", you mean commands that someone could do in Github to cause these to run, I'm indifferent.  If we combine them, then we just have to document that certain failures (`pyright_strict`) are OK, while the others should be fixed sooner than later.

Reply:
Seems like it might be better if we keep them separated

Reply:
I am going to try and make a mapping so that we can simplify the code, with good docs it will be easier than just a lot of repetition.

Reply:
Sorry @twoertwein for the delay i was moving cross country.
I did concentrate all the actions with one code and a selector as a dictionary that will select the command to run according to the comment, let me know if this is what you intended and I will write the documentation.

Reply:
Looks great! Can you please mention the three commands here https://github.com/pandas-dev/pandas-stubs/blob/main/docs/tests.md ? It can probably integrated in the optional test section (or be a new section).